packages/editor/src/components/CodeEditor/CssEditor.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useRef } from 'react';
 import CodeMirror from 'codemirror';
-import { Box } from '@chakra-ui/react';
+import { Box } from '@sunmao-ui/editor-sdk';
```

Comment:
I think we should not import `Box` and other chakra component from `editor-sdk`. Becase they are not widget, just common ui components.
And I don't think we will update these components in `editor-sdk`. 
Importing components from 2 packages is confusing.

Reply:
But the `editor-sdk` should provide the same base editor UI components for developing widgets. Otherwise, the 
 widget developers need to use chakra UI in their projects.

Maybe change to `import { Box } from '@sunmao-ui/edtior-sdk/ui` could make it clear?

Reply:
> Maybe change to import { Box } from '@sunmao-ui/edtior-sdk/ui could make it clear?
That will not help either. Since we don't have better idea, don't change it.